src/ui/src/components/shared/SharedJsonViewer/SharedJsonViewer.vue

```diff
@@ -55,6 +61,7 @@ import SharedJsonViewerObject from "./SharedJsonViewerObject.vue";
 import SharedJsonViewerValue from "./SharedJsonViewerValue.vue";
 import SharedJsonViewerChildrenCounter from "./SharedJsonViewerChildrenCounter.vue";
 import SharedControlBar from "../SharedControlBar.vue";
+import WdsCopyClipboardButton from "@/wds/WdsCopyClipboardButton.vue";
```

Comment:
Since this component imports floating UI, it's better to make it lazy to split the bundle.
```suggestion
import { defineAsyncComponentWithLoader } from "@/utils/defineAsyncComponentWithLoader";

const WdsCopyClipboardButton = defineAsyncComponentWithLoader({
	loader: () => import("@/wds/WdsCopyClipboardButton.vue"),
});
```